### ISSUE TITLE
mac version doesn't open advanced GUI in feature selection

### ISSUE THREAD

**Issue by arobie** [kristinbranson/JAABA#303]
The feature selection GUI doesn't seem to work in advance mode on a mac.

**Comment by mkabra** [comment]
Seems to work fine on the example I tried. Is it always not working or doesn't work for a particular animal? (flyhoghof maybe?).

**Comment by arobie** [comment]
Yeah - I tried it on my mac too with the same target type in case that was the problem and I got the advanced feature editting GUI to open without a problem. Any suggestions on how to troubleshoot why its not work on steffi's laptop? She has the same retina pro that I do.

**Comment by mkabra** [comment]
Is the Matlab version same? Does the problem still persist when she restarts her laptop?
